### PR TITLE
[PREGEL] Add loading state

### PR DESCRIPTION
**Pull request by jvolmer**
The loading state is added in PR https://github.com/arangodb/arangodb/pull/16539.

Upstream:
- https://github.com/arangodb/arangodb/pull/16595 (3.11) -  do not commit generated examples for this version
- https://github.com/arangodb/arangodb/pull/16759 (3.10)

### CODE REVIEW

**Review by markuspf — APPROVED** [review]
LGTM

**Comment by Simran-B** [comment]
Needs to be reflected in https://github.com/arangodb/arangodb/blob/devel/Documentation/DocuBlocks/Rest/Pregel/get_api_control_pregel_status.md

**Comment by jvolmer** [comment]
@Simran-B Adaption of docu of REST Endpoint https://github.com/arangodb/arangodb/pull/16595

File: 3.10/graphs-pregel.md [hunk 1]
```diff
@@ -99,12 +99,15 @@ The `state` field has one of the following values:
...
-| `"recovering"` | The execution is actively recovering, will switch back to "running" if the recovery was successful
-| `"canceled"`   | The execution was permanently canceled, either by the user or by an error.
 | `"storing"`    | The algorithm finished, but the results are still being written back into the collections. Occurs if the `store` parameter is set to `true` only.
 | `"done"`       | The execution is done. In version 3.7.1 and later, this means that storing is also done. In earlier versions, the results may not be written back into the collections yet. This event is announced in the server log (requires at least *info* log level for the *pregel* topic).
+| `"canceled"`   | The execution was permanently canceled, either by the user or by an error.
+| `"in error"`   | The execution is in an error state. This can be caused by primary DB-Servers being not reachable or being non responsive. The execution might recover later, or switch to "canceled" if it was not able to recover successfully
+| `"recovering"` | The execution is actively recovering, will switch back to "running" if the recovery was successful
```
Comment (Simran-B) [inline 3.10/graphs-pregel.md:109]:
```suggestion
| `"recovering"` | The execution is actively recovering and switches back to "running" if the recovery is successful
```

**Review by Simran-B — APPROVED** [review]

File: 3.10/graphs-pregel.md [hunk 2]
```diff
@@ -103,14 +98,13 @@ The `state` field has one of the following values:
 | State          | Description    |
 |:---------------|:---------------|
 | `"running"`    | Algorithm is executing normally.
-| `"in error"`   | The execution is in an error state. This can be caused by primary DB-Servers being not reachable or being non responsive. The execution might recover later, or switch to "canceled" if it was not able to recover successfully
```
Comment (ansoboleva) [inline 3.10/graphs-pregel.md:106]: @Simran-B Julia has added three more states (loading, fatal_error, none). Looks like they were removed during your review. Please add them back.

**Review by ansoboleva — CHANGES_REQUESTED** [review]
please see my comment about missing states

File: 3.10/graphs-pregel.md (hunk 2 above)
Comment (Simran-B) [reply 3.10/graphs-pregel.md:106]: Sorry @jvolmer, the visual 3-way merge in VS Code confused me big time. Restored.

**Review by ansoboleva — APPROVED** [review]

**Pull request merged**